code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -255,21 +255,28 @@
 		src << "<span class='warning'>You must have an aggressive grab to gut your prey!</span>"
 		return
 
-	last_special = world.time + 25
-
-	visible_message("<span class='warning'><b>\The [src]</b> rips viciously at \the [G.affecting]'s flesh with its mandibles!</span>")
-
 	if(istype(G.affecting,/mob/living/carbon/human))
 		var/mob/living/carbon/human/H = G.affecting
-		H.apply_damage(25,BRUTE, sharp=1, edge=1)
-		msg_admin_attack("[key_name_admin(src)] mandible'd [key_name_admin(H)] (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[src.x];Y=[src.y];Z=[src.z]'>JMP</a>)",ckey=key_name(src),ckey_target=key_name(H))
+		var/mob/living/carbon/human/H2 = src
```

Comment:
uh, wot? `src` is already of type `/mob/living/carbon/human`, why do you need to restate it as being that?